doc/nrf/releases_and_maturity/migration/migration_guide_3.2.rst

```diff
@@ -200,7 +200,79 @@ Libraries
 
 This section describes the changes related to libraries.
 
-|no_changes_yet_note|
+PDN library
+-----------
+
+.. toggle::
+
+   * The :ref:`pdn_readme` library has been deprecated in favor of the PDN management functionality provided by the :ref:`lte_lc_readme` library and will be removed in a future |NCS| release.
+
+     To migrate your application to use LTE link control PDN management instead of the PDN library, complete the following steps:
+
+     #. Kconfig options:
+
+        * Replace:
+
+           * The ``CONFIG_PDN`` Kconfig option with the :kconfig:option:`CONFIG_LTE_LC_PDN_MODULE` Kconfig option.
```

Comment:
Do we not want to mention CONFIG_LTE_LINK_CONTROL which is also needed although many apps already have it enable if they used PDN library.